EIPS/eip-4804.md

```diff
@@ -53,7 +53,7 @@ where
 
 ### Resolve Mode
 
-Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode" method. The protocol currently supports two resolve modes:
+Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode() returns (bytes32)" method. The protocol currently supports two resolve modes:
 
 #### Manual Mode
 
```

Comment:
Thanks for the comment.  The original interpretation means path ["?" query] to calldata, but definitively we can extend it to MIME.

Reply:
To be sure : 
The meaning I want to convey is : Since the manual mode will not do any interpretation of **path** and **query**, the users can write anything they want, but I want to remind that the browser will still interpret the [.<extension>] part of the path as MIME and set the content-type header accordingly.
If you believe it is not necessary to repeat it, then I'll trust your judgment.

Reply:
Good suggestion!  Let me put the better wording here.